Modules/_tkinter.c

```diff
@@ -3389,6 +3389,14 @@ DisableEventHook(void)
 #endif
 }
 
+static void
+module_free(void *mod)
+{
+    Py_DECREF(Tkinter_TclError);
+    Py_DECREF(Tkapp_Type);
+    Py_DECREF(Tktt_Type);
+    Py_DECREF(PyTclObject_Type);
```

Comment:
The free callback usually uses `Py_CLEAR` instead of `Py_DECREF` by the way.

Reply:
Oh right, just added it.